tests/Attributes/Fixtures/Basic/Controller.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tobion\OpenApiSymfonyRouting\Tests\Attributes\Fixtures\Basic;
+
+use OpenApi\Attributes as OAT;
```

Comment:
please use only `OA` as name to be consistent with the annotations and swagger-php doc